tests/jenkins/TestRunNonSecurityPerfTestScript.groovy

```diff
@@ -152,34 +143,34 @@ class TestRunNonSecurityPerfTestScript extends BuildPipelineTest {
     }
 
     @Test
-    void testRunSecurityPerfTestScript_verifyJob_aborted() {
+    void testRunSecurityPerfTestScript_verifyJob_aborted() throws Exception{
         binding.setVariable('BUNDLE_MANIFEST', 'tests/jenkins/data/opensearch-1.3.0-bundle.yml')
         binding.setVariable('HAS_SECURITY', true)
+        helper.registerAllowedMethod("cfnDescribe", [Map.class]) { args -> return null}
         helper.registerAllowedMethod('sh', [String.class], { String cmd ->
             updateBuildStatus('ABORTED')
         })
         runScript("jenkins/opensearch/perf-test.jenkinsfile")
-        
+
         assertJobStatusAborted() 
         assertCallStack()
-        assertCallStack().contains("perf-test.cfnDescribe({stack=test-single-security-1236-x64-perf-test})")
-        assertCallStack().contains("'test-single-security-1236-x64-perf-test' does not exist, nothing to remove")
+        assertCallStack().contains("cfnDescribe({stack=test-single-security-1236-x64-307})")
     }    
 
     @Test
-    void testRunNonSecurityPerfTestScript_verifyJob_aborted() {
+    void testRunNonSecurityPerfTestScript_verifyJob_aborted() throws Exception{
         binding.setVariable('BUNDLE_MANIFEST', 'tests/jenkins/data/opensearch-1.3.0-non-security-bundle.yml')
         binding.setVariable('HAS_SECURITY', false)
-        helper.registerAllowedMethod("cfnDescribe", [Map]) { args -> return 'anything'}
+        helper.registerAllowedMethod("cfnDescribe", [Map.class]) { args -> return true}
```

Comment:
Can you add a test case for `false` too? It should print ` "Stack '${stackName}' does not exist, nothing to remove"`

Reply:
Thanks, @gaiksaya , added test to assert nothing to delete.